src/integration/java/io/pinecone/integration/inference/EmbedTest.java

```diff
@@ -4,8 +4,8 @@
 import io.pinecone.clients.Pinecone;
 import org.junit.jupiter.api.Assertions;
 import org.junit.jupiter.api.Test;
-import org.openapitools.control.client.ApiException;
-import org.openapitools.control.client.model.EmbeddingsList;
+import org.openapitools.inference.client.ApiException;
```

Comment:
Example of inference being its own module instead of being a part of the control plane which resulted in updating the import statement